etc/emacs-vterm-zsh.sh

```diff
@@ -17,9 +17,7 @@ function vterm_printf(){
 
 # Completely clear the buffer. With this, everything that is not on screen
 # is erased.
-if [[ "$INSIDE_EMACS" = 'vterm' ]]; then
-    alias clear='vterm_printf "51;Evterm-clear-scrollback";tput clear'
-fi
+alias clear='vterm_printf "51;Evterm-clear-scrollback";tput clear'
```

Comment:
I don't think you want this because it will overwrite /usr/bin/clear if you have that. Although `tput clear` might be enough on most systems?